app/src/main/kotlin/com/hippo/ehviewer/Settings.kt

```diff
@@ -117,6 +118,7 @@ object Settings : DataStorePreferences(null) {
     val qSSaveProgress = boolPref("qs_save_progress", true)
     val security = boolPref("require_unlock", false)
     val recentDownloadLabel = stringOrNullPref("recent_download_label", null)
+    var downloadFilterMode = intPref("download_filter_mode", DownloadsFilterMode.Default.flag)
```

Comment:
typo? `val`

Reply:
yes, i fix it [a7930bd](https://github.com/FooIbar/EhViewer/pull/998/commits/a7930bd054f077c5cb154b077d3f563f6330a8d4)